sdk/websites/Microsoft.Azure.Management.WebSites/src/Generated/Models/SlowRequestsBasedTrigger.cs

```diff
@@ -30,11 +30,13 @@ public SlowRequestsBasedTrigger()
         /// Initializes a new instance of the SlowRequestsBasedTrigger class.
         /// </summary>
         /// <param name="timeTaken">Time taken.</param>
+        /// <param name="path">Request Path.</param>
         /// <param name="count">Request Count.</param>
         /// <param name="timeInterval">Time interval.</param>
-        public SlowRequestsBasedTrigger(string timeTaken = default(string), int? count = default(int?), string timeInterval = default(string))
+        public SlowRequestsBasedTrigger(string timeTaken = default(string), string path = default(string), int? count = default(int?), string timeInterval = default(string))
```

Comment:
`string path = default(string)` should be at the end since it is a new parameter.